channeldb/invoices.go

```diff
@@ -329,7 +330,7 @@ func serializeInvoice(w io.Writer, i *Invoice) error {
 	if i.Terms.Settled {
 		settleByte[0] = 1
 	}
-	if _, err := w.Write(settleByte[:]); err != nil {
+	if err := binary.Write(w, byteOrder, settleByte[:]); err != nil {
```

Comment:
We should be able to pass `i.Terms.Settled` directly into the `binary.Write` here, eliminating the declaration of `settleByte` all together.